### ISSUE TITLE
bundle error

### ISSUE THREAD

**Issue by KuteKetX** [rapid7/metasploit-framework#10272]
```
chronos@localhost / $ cd /usr/local/share/metasploit-framework/
chronos@localhost /usr/local/share/metasploit-framework $ ls
app                 COPYING  docker-compose.yml  Gemfile                LICENSE                       msfconsole  msfupdate  README.md  test
CODE_OF_CONDUCT.md  data     Dockerfile          Gemfile.local.example  LICENSE_GEMS                  msfd        msfvenom   script     tools
config              db       documentation       Gemfile.lock           metasploit-framework.gemspec  msfrpc      plugins    scripts    Vagrantfile
CONTRIBUTING.md     docker   external            lib                    modules                       msfrpcd     Rakefile   spec
chronos@localhost /usr/local/share/metasploit-framework $ msfconsole 
Could not find grpc-1.7.2 in any of the sources
Run `bundle install` to install missing gems.
chronos@localhost /usr/local/share/metasploit-framework $ bundle install
Fetching gem metadata from https://rubygems.org/...........
grpc-1.7.2-x86_64-linux requires ruby version < 2.5, which is incompatible with the current version, ruby 2.5.1p57
chronos@localhost /usr/local/share/metasploit-framework $ 
```

## System stuff
Model: Acer Chromebook 11
```
chronos@localhost / $ screenfetch
/usr/local/bin/screenfetch: line 1341: [: =: unary operator expected
             .,:loool:,.               chronos@localhost
         .,coooooooooooooc,.           OS: Chrome OS 
      .,lllllllllllllllllllll,.        Kernel: x86_64 Linux 4.4.138-14463-ga3af6d2337eb
     ;ccccccccccccccccccccccccc;       Uptime: 3d 6h 37m
   'ccccccccccccccccccccccccccccc.     Packages: 90
  ,ooc::::::::okO0000OOkkkkkkkkkkk:    Shell: bash 4.3.48
 .ooool;;;;:xK0kxxxxxk0XK0000000000.   CPU: Intel Celeron N2840 @ 2x 2.5823GHz
 :oooool;,;OKdddddddddddKX000000000d   GPU: 
 lllllool;lNdllllllllllldNK000000000   RAM: 2988MiB / 3865MiB
 llllllllloMdcccccccccccoWK000000000  
 ;cllllllllXXc:::::::::c0X000000000d  
 .ccccllllllONkc;,,,;cxKK0000000000.  
  .cccccclllllxOOOOOOkxO0000000000;   
   .:cccccccclllllllloO0000000OOO,    
     ,:ccccccccclllcd0000OOOOOOl.     
       '::cccccccccdOOOOOOOkx:.       
         ..,::ccccxOOOkkko;.          
             ..,:dOkxl:.    
```
### Metasploit version
latest version as of July 7, 2018 (i cant check the version lol)
### I installed Metasploit with:
chromebrew  (https://github.com/skycocker/chromebrew/)

**Comment by bcoles** [comment]
Are you still experiencing this issue?

It should be fixed, as grpc now supports Ruby 2.5; as per:

* https://github.com/grpc/grpc/issues/13903
* https://github.com/grpc/grpc/issues/13908
* https://github.com/grpc/grpc/issues/13908

**Comment by busterb** [comment]
Thanks for the note, closing this one out for now.
